src/components/layerselector/style/layerselector.less

```diff
@@ -0,0 +1,138 @@
+.bgLayerDefault {
+  width: 96px;
+  height: 64px;
+  left: 10px;
+  position: absolute;
+  background-color: rgba(255, 255, 255, 0.9);
+  -webkit-box-shadow: 6px 6px 12px rgba(0, 0, 0, 0.175);
+  box-shadow: 6px 6px 12px rgba(0, 0, 0, 0.175);
+  padding: 3px;
+  overflow: hidden;
+  z-index: 1000;
+  cursor: pointer;
+
+  @media (max-width: @screen-tablet) {
+    width: 48px;
+    height: 32px;
+  }
+  
+  @media (max-width: @screen-phone) {
+    width: 48px;
+    height: 32px;
+  }
+}
+
+.bgLayer0 {
```

Comment:
Regarding custom class names convention is: prefix with ga and don't use camel case unsless it's an id.
`ga-import-wms-container` for instance.